definition/abilities.md

```diff
@@ -1,44 +1,73 @@
-<!-- When adding abilities, check if they make sense for: A Human, our agent, and a LLM  -->
-
-Ability
-- To perceive
-    - To simplify perceived entities or their attributes into single simpler ones
-    - To see>
-        - To read text
-    - To hear
-    - To be able to identify and abstract any object or concept
-
-- To make decisions
-    - Association cause -> effect
-    - To reason using logic
-    - To do inner simulations
-        - To plan
-        - To do mental logic processes like comparison
-
-- To communicate
-    - To react accordingly to the sentence heard (IN)
-        <!-- (This can be an instruction, a question, etc.) -->
-    - To communicate ideas (OUT)
-        - To ask questions (IN)
-        - To answer questions (OUT)
-    - To relate words with the concept they represent
-        <!-- This ability is crucial to truly understand any sentence -->
-- To learn
-    - To learn autonomously (Trial and error, experiment)
-        - To infere knowledge
-        - To manage new knowledge
-
-- To act
-    - To define act functions (Act solidification)
-    - To follow instructions/algorithms/protocols
-
-- To understand and remember
-    - To understand time
-        - Sequenciation
-        - Timing of actions
-        - Time linked events
-    - To understand space
-    - To store semantinc memories
-    - To store episodic memories
-    - To manage temporal and long term memory
-    - To be able to store, abstract and categorize any type of knowledge
\ No newline at end of file
+<!-- 
+When adding an ability, check if 
+- it makes sense for a human, symbolic agent, and an LLM 
+- a regular person would say that they have this ability if asked
+- you are adding it in a correct place, make sure the children are extensions/specifications of the parent. Remember that this is not a tree of dependencies - don't add children abilties because they are required to do the parent ability. For example `ability to store produral knowledge` should not be under the `ability to act`, it is indeed required to act, but it's not an expansion of the `ability to act`.
+
+tags:
+- `explain` - please provide an example in the PR comment
+- `invalid` - the entry is not formulated as an ability
+- `remove` - the entry is not needed because it is invalid or unclear,it needs to be deleted or replaced
+
+It's a good idea to reolve all the tags before merging a PR.
+-->
+
+# Ability List
+
+Ability ...
+
+- 1\. to perceive
+  - 1.1 [explain] to simplify perceived entities or their attributes into single simpler ones
+  - 1.2 to see
+    - 1.2.1 to read text
+  - 1.3 to hear
+  - 1.4 [explain] to be able to identify and abstract any object or concept
+
+- 2\. to make decisions
+  - [explain,invalid] Association cause -> effect
+  - to reason using logic
+  - to do simulate the real world
+  - to plan
+    - [explain] to do mental logic processes
+      - [explain] to compare something
+
+- 3\. to communicate
```

Comment:
Communication sounds like an externally directed action, so I guess it would make sense to move it to be part of the `to act` ability.